app/views/jobs/_form.html.erb

```diff
@@ -1,15 +1,30 @@
+<div class="form-group">
+   <% if @job.errors.any? %>
```

Comment:
this piece of error code is reused (mostly) - can we move it to a partial to reuse it?